### PR TITLE
add r-biodivmapr

### PR DESCRIPTION
**Pull request by bgruening**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bgruening** [comment]
Will be fixed with https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/340 - I hope.

File: recipes/r-biodivmapr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    - r-rgeos
+
+test:
+  commands:
+    - $R -e "library('biodivMapR')"           # [not win]
+    - "\"%R%\" -e \"library('biodivMapR')\""  # [win]
+
+about:
+  home: https://github.com/jbferet/biodivMapR
+  license: GPL-3.0-or-later
```
Comment (cbrueffer) [inline recipes/r-biodivmapr/meta.yaml:83]:
```suggestion
  license: GPL-3.0-only
```

File: recipes/r-biodivmapr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  summary: 'An R package for α- and β-diversity mapping using remotely-sensed images'
+  license_family: GPL3
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - bgruening
```
Comment (cbrueffer) [inline recipes/r-biodivmapr/meta.yaml:91]: You're still part of `conda-forge/r`, seems redundant?

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**